Mathlib/Algebra/BigOperators/Finsupp.lean

```diff
@@ -107,6 +107,13 @@ theorem prod_ite_eq [DecidableEq α] (f : α →₀ M) (a : α) (b : α → M 
 #align finsupp.prod_ite_eq Finsupp.prod_ite_eq
 #align finsupp.sum_ite_eq Finsupp.sum_ite_eq
 
+@[to_additive (attr := simp)]
```

Comment:
Is it a good `simp` lemma? Also, you rewrite many `ite`s as `if .. then .. else ..` but use `ite` here.

Reply:
Yes, the `ite -> if ... then ... else ...` is my mistake. About the `simp` attribute, I just copied the definition above for `prod_ite_eq` without much thought. But now that I think about it, since the lemma requires a iff hypothesis `(h : ∀ x ∈ f.support, p x ↔ a = x)`, it won't ever be triggered by `simp` I think, or at least very very rarely. So I will remove the [simp].